apps/explorer/src/ui/InternalLink.tsx

```diff
@@ -39,12 +39,20 @@ function createInternalLink<T extends string>(
 export const EpochLink = createInternalLink('epoch', 'epoch');
 export const CheckpointLink = createInternalLink('checkpoint', 'digest', formatAddress);
 export const CheckpointSequenceLink = createInternalLink('checkpoint', 'sequence');
-export const AddressLink = createInternalLink('address', 'address', (addressOrNs) => {
+
+export const ObjectLink = createInternalLink('object', 'objectId', formatAddress);
+export const TransactionLink = createInternalLink('txblock', 'digest', formatDigest);
+export const ValidatorLink = createInternalLink('validator', 'address', formatAddress);
+
+const AddressLinkComponent = createInternalLink('address', 'address', (addressOrNs) => {
 	if (isSuiNSName(addressOrNs)) {
 		return addressOrNs;
 	}
 	return formatAddress(addressOrNs);
 });
-export const ObjectLink = createInternalLink('object', 'objectId', formatAddress);
-export const TransactionLink = createInternalLink('txblock', 'digest', formatDigest);
-export const ValidatorLink = createInternalLink('validator', 'address', formatAddress);
+
+export function AddressLink(props: BaseInternalLinkProps & { address: string }) {
```

Comment:
What kind of API volume does thrust generate? We explicitly didnt do this because of that.

Reply:
update to only resolve in places within TXN page